packages/gatsby-plugin-mdx/gatsby/source-nodes.js

```diff
@@ -90,7 +90,7 @@ module.exports = (
     debug(`required`, plugin)
     if (_.isFunction(requiredPlugin.setParserPlugins)) {
       for (let parserPlugin of requiredPlugin.setParserPlugins(
-        plugin.pluginOptions
+        plugin.options
```

Comment:
Let's ensure it's not undefined so plugins don't have to check that
```suggestion
        plugin.options || {} 
```
( Same as in https://github.com/gatsbyjs/gatsby/blob/b4e3c5eb0192832403beebad5a44db08efbfa0ca/packages/gatsby-plugin-mdx/utils/get-source-plugins-as-remark-plugins.js#L67 )

Reply:
Good call, I've actually just run into this issue a few days ago https://github.com/gatsbyjs/gatsby/issues/18135#event-2688457725